buildsettings.json

```diff
@@ -1,7 +1,7 @@
 {
     "version": "1.8.1",
     "avi": {
-        "maxVersion": "21.1.3",
+        "maxVersion": "21.1.4",
```

Comment:
is it 21.1.4 or 21.1.5?

Reply:
we have not decided max version of AKO for 1.8.1. So kept it same as for 1.7.2.